static/js/modules/tables.js

```diff
@@ -0,0 +1,163 @@
+      datum.cycles
+    );
+    return '?cycle=' + _.max(cycles);
+  } else {
+    return '';
+  }
+}
+
+var candidateColumns = [
+  {
+    data: 'name',
+    render: function(data, type, row, meta) {
+      var anchor = $('<a>');
+      anchor.addClass('single-link');
+      anchor.attr('title', data);
+      anchor.attr('data-category', 'candidate');
+      anchor.attr('href', '/candidate/' + row.candidate_id + buildCycle(row));
+      anchor.text(data);
+      return anchor[0].outerHTML;
+    }
+  },
+  {data: 'office_full'},
+  {
+    data: 'election_years',
+    render: function(data, type, row, meta) {
+      return yearRange(data[0], row.active_through);
+    }
+  },
+  {data: 'party_full'},
+  {data: 'state'},
+  {data: 'district'},
+];
+
+var committeeColumns = [
+  {
+    data: 'name',
+    render: function(data, type, row, meta) {
+      var anchor = $('<a>');
+      anchor.addClass('single-link');
+      anchor.attr('title', data);
+      anchor.attr('data-category', 'committee');
+      anchor.attr('href', '/committee/' + row.committee_id + buildCycle(row));
+      anchor.text(data);
+      return anchor[0].outerHTML;
+    }
+  },
+  {data: 'treasurer_name'},
+  {data: 'state'},
+  {data: 'party_full'},
+  {data: 'organization_type_full'},
+  {data: 'committee_type_full'},
+  {data: 'designation_full'}
+];
+
+function mapSort(order, columns) {
+  return _.map(order, function(item) {
+    var name = columns[item.column].data;
+    if (item.dir === 'desc') {
+      name = '-' + name;
+    }
+    return name;
+  });
+}
+
+function mapResponse(response) {
+  return {
+    recordsTotal: response.pagination.count,
+    recordsFiltered: response.pagination.count,
+    data: response.results
+  };
+}
+
+function initTable(table, form, baseUrl, columns) {
+  var draw;
+  var api = table.DataTable({
+    serverSide: true,
+    searching: false,
+    columns: columns,
+    lengthMenu: [30, 50, 100],
+    language: {
+      lengthMenu: 'Results per page: _MENU_'
+    },
+    dom: '<"results-info meta-box results-info--top"lfrip>t<"results-info meta-box"ip>',
+    ajax: function(data, callback, settings) {
+      var api = this.api();
+      var filters = form.serializeArray();
+      parsedFilters = mapFilters(filters);
+      var query = $.extend(
+        {
+          per_page: data.length,
+          page: Math.floor(data.start / data.length) + 1,
+          api_key: API_KEY
+        },
+        parsedFilters
+      );
+      query.sort = mapSort(data.order, columns);
+      $.getJSON(
+        URI(API_LOCATION)
+        .path([API_VERSION, baseUrl].join('/'))
+        .query(query)
+        .toString()
+      ).done(function(response) {
+        callback(mapResponse(response));
+      });
+    }
+  });
+  form.submit(function(event) {
+    event.preventDefault();
+    api.ajax.reload();
+  });
+}
+
+module.exports = {
+  init: function() {
+    var table = $('#results');
```

Comment:
Your call. I tried not to change any of the markup in this patch--`results` happened to be the ID given to the tables previously. I'd be in favor of something more explicit, like `js-results` or `datatable-results`.